src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,35 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+
+interface TodoInfoInt {
+  todo: Todo
+}
+
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+}
+
+export const TodoInfo = ({ todo }: TodoInfoInt) => {
+  const getUserById = (id: number): User => {
+    return usersFromServer.find((user) => user.id === id) as User;
+  };
+
+  return (
+
+    <article
+      key={todo.id}
+      data-id={todo.id}
+      className={todo.completed ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'}
```

Comment:
It's a perfect case to use `classNames` 

```suggestion
      className={classNames('TodoInfo', {
        'TodoInfo--completed': todo.completed,
      })}
```

Reply:
hm, i am not sure that I understand the difference

could you send me a few links to the resources about the difference in using my approach and classnames?

just want to make sure that I understand what's happening here

Reply:
There is an article about classnames: [link](https://medium.com/2359media/why-react-classnames-is-a-lifesaver-and-how-you-should-use-it-if-you-arent-already-b25a44ed5603) - about general advantages

I'd say with `classNames` it's more clear (visually) that `TodoInfo` class exists always and `--completed` is conditional
But it's not critical here